framework/core/Constants.pm

```diff
@@ -256,6 +256,22 @@ our $TAG_FIXED            = "FIXED_VERSION";
 our $TAG_BUGGY            = "BUGGY_VERSION";
 our $TAG_PRE_FIX          = "PRE_FIX_REVISION";
 
+# Filenames for bugs csv files
+our $BUGS_CSV_ACTIVE = "active-bugs.csv";
+our $BUGS_CSV_DEPRECATED = "deprecated-bugs.csv";
+
+# Columns in active-bugs and deprecated-bugs csvs
+our $BUGS_CSV_BUGID = "bug_id";
+our $BUGS_CSV_COMMIT_BUGGY = "revision_id_buggy";
+our $BUGS_CSV_COMMIT_FIXED = "revision_id_fixed";
+our $BUGS_CSV_ISSUE_ID = "bug_report_id";
+our $BUGS_CSV_ISSUE_URL = "bug_report_url";
+our $BUGS_CSV_DEPRECATED_WHEN = "deprecated_version";
+our $BUGS_CSV_DEPRECATED_WHY = "deprecated_reason";
+
+# Reasons for deprecation
+our $DEPRECATED_JVM8 = "JVM8-NonReproducable";
```

Comment:
There is another reason for deprecation: a few bugs were duplicates. Also, it might be worth separating non-reproducible from non-compilable bugs. Reproducable should be Reproducible.

Reply:
I have updated and added the following:
`our $DEPRECATED_DUPLICATE = "Duplicate";`
`our $DEPRECATED_JVM8_REPRO = "JVM8_Not_Reproducible";`
`our $DEPRECATED_JVM8_COMPILE = "JVM8_Does_Not_Compile";`

I changed Closure 63 and 93 to `Duplicate`. Were there any other duplicates?